### PR TITLE
Remove cfitsio as it is now in conda-forge

### PR DESCRIPTION
**Pull request by saimn**
@mwcraig - Is there a way to copy cfitsio from conda-forge, to make it available from the OpenAstronomy channel ?

### CODE REVIEW

**Comment by bsipocz** [comment]
@saimn - I believe that the machinery is the same as in conda-channel-astropy, so if something is in conda-forge we copy it over.

**Comment by saimn** [comment]
@bsipocz - But how is determined what must be copied ? You mean if there is an entry in ` requirements.yml ` ?

**Comment by bsipocz** [comment]
yes, there should be an entry in requirements, and if it's exists in conda-forge, it gets copied over. I'm not sure what happens if there is a recipe template as well (I assume the copying is higher priority), @mwcraig knows the details.

**Comment by mwcraig** [comment]
Recipes are created in this priority order:

+ If a recipe template is present, it is used.
+ If there is no recipe template and the package is on conda-forge, the recipe is is copied from conda-forge.
+ Otherwise, a recipe is generated with `conda skeleton`.

The entire infrastructure really needs a re-write so that building is done on conda-forge and these "subset" channels  use something like `requirements.yml` to mirror those packages. One of these weekends...

**Comment by saimn** [comment]
Thanks for the explanations @mwcraig , I added cfitsio to the `requirements.yml` so it should now be copied from conda-forge.

**Comment by mwcraig** [comment]
Just pushed a change that should make the lights green (updating sunpy version)

**Comment by mwcraig** [comment]
I'm going to have to try doing a real fix for copying from conda-forge; it is failing because right now I just copy the recipe file. The build fails here because there is a patch in the conda-forge repo that isn't being copied. This weekend, I hope...

**Comment by mwcraig** [comment]
Hopefully this builds...

@bsipocz -- if this works and is merged then there will be no need to list version numbers for packages on conda-forge. This will simply pull in whatever the conda-forge version is at the time the builder is run.

**Comment by bsipocz** [comment]
@mwcraig - Sounds great.

**Comment by mwcraig** [comment]
close/reopen to retrigger builds...

**Comment by mwcraig** [comment]
I'll add a restriction on the numpy version tomorrow to exclude numpy 1.12 and then linux/mac should work. 

Hopefully windows will be an easy fix, will have time to boot up a vm on Monday.

File: appveyor.yml [hunk 1]
```diff
@@ -73,4 +75,4 @@ test_script:
     # Generate recipes...
     - "%CMD_IN_ENV% extrude_recipes requirements.yml"
     # Build, and, if necessary, upload.
-    - if exist recipes %CMD_IN_ENV% conda build-all recipes --inspect-channels %DESTINATION_CONDA_CHANNEL% conda-forge openastronomy astropy --matrix-condition="python %PYTHON_BUILD_RESTRICTIONS%"  %UPLOAD%
+    - if exist recipes %CMD_IN_ENV% conda build-all recipes --inspect-channels %DESTINATION_CONDA_CHANNEL% conda-forge openastronomy astropy --matrix-conditions "python %PYTHON_BUILD_RESTRICTIONS%" "%NUMPY_BUILD_RESTRICTIONS%" %UPLOAD%
```
Comment (bsipocz) [inline appveyor.yml]: don't use `"` around ``NUMPY_BUILD_RESTRICTIONS``, as it doesn't work when it's set to `""` above.

**Comment by mwcraig** [comment]
Thanks, @bsipocz -- I've ended up cutting out builds to try to get this to pass (think it is there now). Really need to write a script to just copy from one channel to another tomorrow...

**Comment by mwcraig** [comment]
Merging before something breaks... 😬

**Comment by saimn** [comment]
Thanks @mwcraig !

**Pull request merged**